### DISCUSSION TITLE
Drawer accessibility

### DISCUSSION THREAD

**Discussion by noahsb** [mantinedev/mantine#3016] [Q&A] [answered]
Wanted to create a discussion to follow up on #2972 in case it was missed

**Comment by rtivital** [comment, marked as answer]
If you want, you can contribute this feature. I do not have capacity to work on it now.

**Reply by noahsb** [reply]
No problem! I wanted to understand if there was a reason they have differing APIs

Would you be open to me updating Drawer to match Modal so that the aria-labeledby and aria-describedby are applied automatically?

**Reply by rtivital** [reply]
yes
